modules/imaging_browser/php/viewsession.class.inc

```diff
@@ -221,11 +221,11 @@ class ViewSession extends \NDB_Form
             $SeriesNumber        = $FileObj->getParameter('series_number');
             $SeriesUID           = $FileObj->getParameter('series_instance_uid');
             $EchoTime            = number_format(
-                (float)$FileObj->getParameter('echo_time') * 1000,
+                (int)$FileObj->getParameter('echo_time') * 1000,
```

Comment:
I think this is a step backwards from https://github.com/aces/Loris/pull/5401/files

What is the reason for this change?